includes/helper/class-wp-job-manager-helper-api.php

```diff
@@ -41,13 +41,13 @@ public static function instance() {
 	/**
 	 * Checks if there is an update for the plugin using the WPJobManager.com API.
 	 *
-	 * @deprecated $$next-version$$ Use WP_Job_Manager_Helper_API::bulk_update_check() instead.
+	 * @deprecated 1.42.0 Use WP_Job_Manager_Helper_API::bulk_update_check() instead.
 	 *
 	 * @param array $args The arguments to pass to the endpoint.
 	 * @return array|false The response, or false if the request failed.
 	 */
 	public function plugin_update_check( $args ) {
-		_deprecated_file( __METHOD__, '$$next-version$$', 'WP_Job_Manager_Helper_API::bulk_update_check' );
+		_deprecated_file( __METHOD__, '1.42.0', 'WP_Job_Manager_Helper_API::bulk_update_check' );
```

Comment:
Uh..I'm pretty sure we should use `_deprecated_function` here instead of `_deprecated_file`: https://developer.wordpress.org/reference/functions/_deprecated_function/

```suggestion
		_deprecated_function( __METHOD__, '1.42.0', 'WP_Job_Manager_Helper_API::bulk_update_check' );
```